lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -302,6 +314,44 @@ private JWEObject signEncryptJar(
         return AuthorizationRequestHelper.createJweObject(rsaEncrypter, signedJWT);
     }
 
+    private EvidenceRequest getEvidenceRequestForF2F(List<SignedJWT> credentials)
+            throws UnknownEvidenceTypeException, ParseException {
+        Gpg45Scores gpg45Scores = gpg45ProfileEvaluator.buildScore(credentials);
+        List<List<Gpg45Scores.Evidence>> requiredEvidences =
+                gpg45ProfileEvaluator.calculateEvidencesRequiredToMeetAProfile(
+                        gpg45Scores, ACCEPTED_PROFILES);
+
+        if (requiredEvidences == null) {
+            return null;
+        }
+
+        OptionalInt minViableStrengthOpt =
+                requiredEvidences.stream()
+                        .filter(subList -> subList.size() == 1)
```

Comment:
(I think you'll also need to check we meet the fraud/activity/verification scores here, once we have them available)

Reply:
I think when I'm creating the list of required evidences I'll exclude cases where evidence cannot meet a profile because of the verif/fraud/activity scores - that should mean we won't need to check here

Reply:
I wonder if that's less flexible in the future though?

E.g. if we need to ask 'what verification score do we need to meet a profile' - we could use the same base method and filter for those that only have a nonzero verification score.

Maybe YAGNI